cpp/src/barretenberg/honk/composer/ultra_composer.cpp

```diff
@@ -202,7 +201,7 @@ std::shared_ptr<typename Flavor::ProvingKey> UltraHonkComposerHelper_<Flavor>::c
     // Initialize proving_key
     // TODO(#392)(Kesha): replace composer types.
     proving_key = initialize_proving_key<Flavor>(
-        circuit_constructor, crs_factory_.get(), minimum_circuit_size, num_randomized_gates, ComposerType::PLOOKUP);
```

Comment:
I removed the need to pass this argument in so many places. It's only currently needed for Plonk.